src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -705,11 +705,11 @@ export function selectModalityIcon(appointment) {
 
   let icon = 'fa-blank';
```

Comment:
Is this default still valid?

Reply:
@vbahinwillit, Thanks for reviewing. I change the icon default assignment. Would you please re-review?